types/sinon/sinon-tests.ts

```diff
@@ -181,8 +181,8 @@ function testClock() {
     clock.setImmediate(fn);
     clock.requestAnimationFrame(fn);
 
-    now = clock.setTimeout(fnWithArgs, 0, 1234, "abc");
-    now = clock.setInterval(fnWithArgs, 0, 1234, "abc");
+    clock.setTimeout(fnWithArgs, 0, 1234, "abc");
```

Comment:
the assignment was causing it to test that the return type matches the other functions

Reply:
It doesn't make sense to assign the return value of `setTimeout` to `now`. They might match structurally but not semantically.

Reply:
it was testing that the two methods returned a matching type. whether its initialised as `now` or not isn't relevant.

make a new variable if you want, or add a `$ExpectType` comment. otherwise we're no longer testing the return type of these two